workflow-v2.5.0/tpl/generate_params.toml

```diff
@@ -0,0 +1,249 @@
+[s3]
+# Your AWS access key. Leave it empty if you want to use IAM credentials.
+accesskey = ""
+# Your AWS secret key. Leave it empty if you want to use IAM credentials.
+secretkey = ""
+# Any S3 region
+region = "us-west-1"
+# Your buckets.
+registry_bucket = "your-registry-bucket-name"
+database_bucket = "your-database-bucket-name"
+builder_bucket = "your-builder-bucket-name"
+
+[azure]
+accountname = "YOUR ACCOUNT NAME"
+accountkey = "YOUR ACCOUNT KEY"
+registry_container = "your-registry-container-name"
+database_container = "your-database-container-name"
+builder_container = "your-builder-container-name"
+
+[gcs]
+# key_json is expanded into a JSON file on the remote server. It must be
+# well-formatted JSON data.
+key_json = '''Paste JSON data here.'''
+registry_bucket = "your-registry-bucket-name"
+database_bucket = "your-database-bucket-name"
+builder_bucket = "your-builder-bucket-name"
+
+[swift]
+username = "Your OpenStack Swift Username"
+password = "Your OpenStack Swift Password"
+authurl = "Swift auth URL for obtaining an auth token"
+# Your OpenStack tenant name if you are using auth version 2 or 3.
+tenant = ""
+authversion = "Your OpenStack swift auth version"
+registry_container = "your-registry-container-name"
+database_container = "your-database-container-name"
+builder_container = "your-builder-container-name"
+
+[builder]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.5.0"
+# limits_cpu = "100m"
+# limits_memory = "50Mi"
+
+[slugbuilder]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.4.1"
+
+[dockerbuilder]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.3.2"
+
+[controller]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.5.1"
+# limits_cpu = "100m"
+# limits_memory = "50Mi"
+
+[slugrunner]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.2.1"
+
+[database]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.2.2"
+# Configure the following ONLY if using an off-cluster PostgreSQL database
+name = "database name"
+username = "database username"
+password = "database password"
+host = "database host"
+port = "database port"
+# limits_cpu = "100m"
+# limits_memory = "50Mi"
+
+[registry]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.2.1"
+# limits_cpu = "100m"
+# limits_memory = "50Mi"
+
+[registry_token_refresher]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v1.0.0"
+# Prefix for the imagepull secret created when using private registry
+secretPrefix = "private-registry"
+# Time in minutes after which the token should be refreshed.
+# Leave it empty to use the default provider time.
+tokenRefreshTime = ""
+# limits_cpu = "100m"
+# limits_memory = "50Mi"
+
+[registry_proxy]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v1.1.0"
+hostPort = 5555
+limits_cpu = "100m"
+limits_memory = "50Mi"
+
+[workflowManager]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.4.1"
+versionsApiURL = "https://versions-staging.deis.com"
```

Comment:
this and below should be changed to "https://versions.deis.com" and "https://doctor.deis.com"